src/cltk/tokenizers/word.py

```diff
@@ -41,6 +41,17 @@ def tokenize_sign(self, text: str, model: object = None):
         """
         pass
 
+    @staticmethod
```

Comment:
Really smart, Clement, how you added this

Reply:
PyCharm gives valuable advice!